scripts/install-packages.ts

```diff
@@ -41,7 +41,7 @@ const handlePackages = async (dir: string): Promise<void> => {
 		const pkgDir = join(dir, pkg);
 		const pkgJsonPath = join(pkgDir, "package.json");
 		if (existsSync(pkgJsonPath)) {
-			const ip = doInstall(pkg, pkgDir);
+			const ip = await doInstall(pkg, pkgDir);
```

Comment:
@kylecarbs you fine with this for now?

Reply:
(below we should remove the `await` if this is good to go.

Reply:
I try to remove it and see what happens in docker.

Reply:
```ERROR Failed to install "common" dependencies {"error":{"killed":false,"code":1,"signal":null,"cmd":"yarn --frozen-lockfile --network-concurrency 1"},"stdout":"[1/4] Resolving packages...\n[2/4] Fetching packages...\ninfo If you think this is a bug, please open a bug report with the information provided in \"/src/packages/app/common/yarn-error.log\".\ninfo Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.\n","stderr":"warning package.json: No license field\nwarning ../../package.json: No license field\nwarning @coder/app-common: No license field\nerror An unexpected error occurred: \"https://registry.yarnpkg.com/@material/layout-grid/-/layout-grid-0.41.0.tgz: ENOENT: no such file or directory, open '/usr/local/share/.cache/yarn/v4/npm-@material-layout-grid-0.41.0-2e7d3be76313e0684d573b10c2c6a88b3230d251/node_modules/@material/layout-grid/.yarn-tarball.tgz'\".\n"}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "packages:install" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
The command '/bin/sh -c yarn --frozen-lockfile' returned a non-zero code: 1```

Reply:
That's half the error msg.

Reply:
Nah, its the full of it. 
Try to have a clean checkup (without the node modules).

Full log:
https://gist.github.com/zerdos/e94dd4867f3b2866b87ca002d218ff82

Reply:
Once we convert to yarn workspaces this should be a non-issue. I'm good w/ it for now.